### ISSUE TITLE
Some issues and random things

### ISSUE THREAD

**Issue by birdybro** [city41/AMMiSTer#9]
Using Windows 11 Build 22621.1105 for all of this testing so far:

* Catalog --> Open Folder doesn't open the folder before the build catalog is made. So I don't know where the files are going to go.
* Can't cancel the build catalog action without closing the whole application.
* File --> Save Plan As... doesn't give response to the user saying that they need to build their catalog/create plan first in order to save it.
* I just downloaded the latest release, did File --> New Plan, and it tells me to "Please update first" in the msg box that pops up. Might be a misattributed response for the user since I think it means that I need to build the catalog first?
* The initial catalog build for the MRAs feels very slow, is there a way to speed it up by multiple downloads in parallel like 4-8 at a time? Just curious, I have no familiarity with anything you built this with, only electron as a user.
* For some reason, if I name a folder `Favorites` under my new plan, it renames it to `favorites` (lowercase f) when I click off of it, but when I go to rename again, it shows capital F. So I think the display of this is showing a bug:
![image](https://user-images.githubusercontent.com/16388068/212797582-e61c5384-629d-4954-826f-71c9bb0dde90.png)
![image](https://user-images.githubusercontent.com/16388068/212797607-be844002-7634-40e3-a297-70b4f0d964b1.png)
* You might benefit from using the arcade organizer's reference DB for the metadata as there are plenty of MRA's that are outdated. @Toryalai1 has done an excellent job making a large spreadsheet of everything used in the arcade organizer script here --> https://github.com/Toryalai1/MiSTer_ArcadeDatabase/blob/main/ArcadeDatabase_CSV/ArcadeDatabase230110.csv. An example of the benefit of using his DB instead is looking at 1941, 1942, and the 1943 games and seeing how Jotego's MRA's currently don't have the category matching the typical format used by arcade-italia and mame xml's.

**Comment by city41** [comment]
Nice feedback, thanks. I fixed the favorites case thing and opening the catalog folder in 0.15.6. I'll pull the rest into their own issues as they are bigger to tackle.

**Comment by city41** [comment]
Oh I changed "Please update first" to "Please build a catalog first" too

**Comment by birdybro** [comment]
Sorry for bundling them together, everyone has a different preference for how to handle this sort of thing. :D
